test/index.js

```diff
@@ -0,0 +1,10 @@
+var expect = require('chai').expect
+var path = require('path')
+
+describe('main.js', function () {
+  it('must export email of the second customer', function(done){
```

Comment:
Nit: since everything in this test is synchronous, we can omit the `done` callback in the function signature (and also, importantly, the call to `done()` in line 8).